src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -327,6 +329,29 @@ public static FormulaValue Text(EvalVisitor runner, EvalVisitorContext context,
             return Text(runner.GetFormattingInfo(), irContext, args, runner.CancellationToken);
         }
 
+        public static FormulaValue TextToBlob(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
+        {
+            runner.CancellationToken.ThrowIfCancellationRequested();
+
+            StringValue sv = (StringValue)args[0];           
+            return new BlobValue(new StringBlob(sv.Value));
+        }
+
+        public static FormulaValue BlobToText(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
+        {
+            if (args[0] is BlobValue bv && bv?.ResourceElement is UriBlob uri)
+            {
+                return FormulaValue.New(uri.Uri.ToString());
+            }
+

```

Comment:
Can we just delete this and always fail. 

Just anxious about Uris in general , and a partial working thing may get people's hopes up.

Reply:
I'll remove UseBlob completely & BlobToText